examples/README.md

```diff
@@ -24,26 +24,26 @@ west init -l digital_microscope
 west update
 ```
 
-This also fetches a zephyr with (currently custom) `robotframework` harness support for twister, and all
-necessary modules
+This also fetches a Zephyr with (currently custom) `robotframework` harness support for Twister, and all
+necessary modules.
 
 ## Creating tests for example
 
-To create robot tests for an example, move into the `assets` folder of the example and call `fret-to-robot`
+To create Robot tests for an example, move into the `assets` folder of the example and call `fret-to-robot`
```

Comment:
Robot Framework (to be consistent with the rest)

Reply:
Thank you, updated!